prac_05/word_occurrences.py

```diff
@@ -0,0 +1,23 @@
+"""
+Word Occurrences:
+Estimate: 30 min
+Actual: 25 min
+"""
+
+word_to_count = {}
+text = input("Text: ")
+
+words = text.split()
+
+for word in words:
+    try:
+        word_to_count[word] += 1
+    except KeyError:
+        word_to_count[word] = 1
+
+words = list(word_to_count.keys())
+words.sort()
+
+max_length = max((len(word) for word in words))
```

Comment:
Do you need to create a list to store the sorted words? Could this be done in your list comprehension statement? :)